src/components/DynamicFormContainer/DynamicForm/DynamicForm.tsx

```diff
@@ -36,16 +36,19 @@ export const DynamicFormRaw: FunctionComponent<DynamicFormProps> = ({
   attachmentAcceptedFormat = "",
 }) => {
   const { data, ownership } = form;
+  const isTransferableRecord = type === "TRANSFERABLE_RECORD";
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const mergeFormValue = (value: any): void => {
     // Avoid using spread which will lazy copy the object
     // See discussion: https://github.com/rjsf-team/react-jsonschema-form/issues/306
     const nextFormData = cloneDeep(data.formData);
+    if (isTransferableRecord) {
+      form.ownership = value?.ownership;
+    }
```

Comment:
this looks suspicious but i don't have a better solution for this

Reply:
ohh okie let me check with raymond😅!

Reply:
Ehhhh `form` refers to the value passes into this function? You shouldn't be manipulating function params directly. There is the `setOwnership` function right?

Reply:
![image](https://user-images.githubusercontent.com/7406870/103052137-5a47c100-45d3-11eb-9136-b2a993b6f5a1.png)

^ you cannot used `setOwnership` because it is fighting with the `setFormData` as well. You need to go to `src/common/context/forms/index.tsx` and fix it from there. Probably want to combine them into a single function `setCurrentForm` with `{form, ownership}` as the param and then either remove `setCurrentFormOwnership` and `setCurrentFileName` or get these 2 functions to call `setCurrentForm`.

In this file you are editing, use the setCurrentForm to set both the data and the ownership in the same call.